src/client/lib/utils.js

```diff
@@ -125,14 +125,13 @@ export var createElement = function(type, attrs={}, text=null) {
 };
 
 
-
-export var toArray = (arrayLike) => {
+export let toArray = (arrayLike) => {
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).